runtime/queries/r/locals.scm

```diff
@@ -3,6 +3,7 @@
 (function_definition) @local.scope
 
 (formal_parameters (identifier) @local.definition)
+(formal_parameters (default_parameter (identifier) @local.definition))
```

Comment:
looks like an unrelated change snuck in here

Reply:
Thanks for spotting! Change has been reverted. It's still in the commit history. Let me know if you'd prefer that I force push changes to the commit history or if squashing on merge is fine.

Reply:
If you like to force-push and constantly keep a clean-looking branch, I don't mind the force-pushes. The squash and merge at the end is also very easy so it's up to you really

Reply:
I usually squash and merge - it feels safer to me not to dabble in force territory.